pyengine/katana.py

```diff
@@ -1,12 +1,22 @@
-def compareprice(side, p1, p2):
+def compareprice(side, p1, p2, q1, q2):
     if side == 'buy':
-        if p1 <= p2:
+        if p1 < p2:
             return True
+        if p1 == p2:
+            if q1 > q2:
+                return True
+            else:
+                return False
         else:
             return False
     elif side == 'sell':
-        if p1 >= p2:
+        if p1 > p2:
             return True
+        if p1 == p2:
+            if q1 > q2:
```

Comment:
what if q1 == q2? question is for both buy and sell versions

Reply:
changing to >=, so we end up using the latest quote if other factors are the same